app/helpers/session.rb

```diff
@@ -0,0 +1,11 @@
+def current_user
+  @current_user ||= User.find_by(id: session[:user_id])
+end
+
+def logged_in?
+  !!user_loggedin
+end
+
+def require_user
+  redirect 'users/login' unless logged_in?
```

Comment:
Heads up: there is no beginning `/` in this path. Thus this path will be relative to the current location, which will  probably not land at `/users/login` like you want.